src/views/certifications/views/landing/components/certification-program-summary-card/certification-program-summary-card.module.css

```diff
@@ -0,0 +1,31 @@
+.splitSection {
+	padding: 8px 8px 0 8px;
+
+	@media (--dev-dot-tablet-up) {
+		padding: 24px 24px 0 24px;
+	}
+}
+
+.heading {
+	composes: hds-typography-display-500 from global;
+	font-weight: var(--token-typography-font-weight-bold);
+	color: var(--token-color-foreground-strong);
+	hyphens: auto;
```

Comment:
![IMG_1945](https://user-images.githubusercontent.com/825855/207892309-4ab76117-7884-401d-827e-e34233843421.png)
![IMG_1946](https://user-images.githubusercontent.com/825855/207892318-961c9129-f3dc-4145-808a-6f976d34c878.png)

Should we be hyphenating headings here?

Reply:
I was on the fence 🤔 I think when I first stubbed this in, I didn't have responsive `font-size` for the headings, so they were overflowing around our smaller mobile viewports, and that felt more broken than hyphens.

But now that we do have a responsive font-size here, it does seem better to prevent hyphens 👍 Have updated to remove `hyphens: auto` - thanks for catching this!